.npmignore

```diff
@@ -5,3 +5,4 @@ tsconfig.json
 src
 test
 .env*
+hardhat.config.ts
```

Comment:
why we ignoring it?

Reply:
-sorry..this was an attempt to fix an issue with tsc not liking hardhat.config.ts being outside of rootDir

-the fix was to remove it from the "files" key of tsconfig (it was only put there as a recommendation by hardhat)

-i will remove it